bugout-dev/drones
# Add deleting logic.

## Changes

Add clean journals service:
Add limit of journals entries per journal.
Working only with humbug journals(`Journal.search_index == None`).




## How to test these changes?

Manualy.

It extend drones cli:

Reports current journal count and expected amount for delete:

```
drones cleanup reports --max-entries <limit-of-entry-in-journal>
```
If amount of entries in journal is 0 then that journal not appear in report. 

Deleting command:
Collect journal_id and amount of entries inside and start deleting with batching logic work with 1 journal at time.
```
drones cleanup journals --max-entries 1000000 --batch-size 10000
```







## Related issues
